pkgs/by-name/cl/claude-squad/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  tmux,
+  gh,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```
Go builder now supports fixed-point lambdas, prefer them over rec